ui/app/components/clients/vertical-bar-chart.js

```diff
@@ -111,18 +121,14 @@ export default class VerticalBarChart extends Component {
       .attr('height', '100%')
       .attr('width', '30px') // three times width
       .attr('y', '0') // start at bottom
-      .attr('x', (data) => xScale(data.month)); // not data.data because this is not stacked data
+      .attr('x', (data) => xScale(data[this.xKey])); // not data.data because this is not stacked data
 
     // MOUSE EVENT FOR TOOLTIP
     tooltipRect.on('mouseover', (data) => {
-      let hoveredMonth = data.month;
-      this.tooltipTotal = `${data.clients} ${data.new_clients ? 'total' : 'new'} clients`;
-      this.uniqueEntities = `${data.entity_clients} unique entities`;
-      this.nonEntityTokens = `${data.non_entity_clients} non-entity tokens`;
-      // let node = chartSvg
-      //   .selectAll('rect.tooltip-rect')
-      //   .filter(data => data.month === this.hoveredLabel)
-      //   .node();
+      let hoveredMonth = data[this.xKey];
```

Comment:
I'm noticing `let` is used for variable assignment in this file but am wondering if any of the variables would ever be reassigned? If not I think `const` would be a better choice to signal that the variables will not be reassigned. This is probably a developer preference type of thing but wanted to offer my thoughts.

Reply:
Nope - i think `const` would be preferable here!